iam/core.py

```diff
@@ -40,6 +40,36 @@ def teardown(bot):
     for handler in LOG.handlers:
         LOG.removeHandler(handler)
 
+async def show_help_all(bot, target):
+    """Send help text for all commands to target.
+
+    Args:
+        bot: Bot to generate help text for.
+        target: Object to send message to.
+    """
+    out = ["**All commands:**"]
```

Comment:
`out = ["**All commands:**"] + [make_help_text(cmd) for cmd in bot.commands if not cmd.hidden]`